svm-transaction/src/svm_message.rs

```diff
@@ -40,6 +39,15 @@ pub trait SVMMessage: Debug {
     /// This does not consider if write-locks are demoted.
     fn num_write_locks(&self) -> u64;
 
+    /// Return an iterator over the instructions in the message, paired with
+    /// the pubkey of the program.
+    fn program_instructions_iter(
+        &self,
+    ) -> impl Iterator<Item = (&Pubkey, SVMInstruction<'_>)> + Clone;
+}
+
+// - Debug to support legacy logging
+pub trait SVMMessage: Debug + SVMStaticMessage {
     /// Return the recent blockhash.
     fn recent_blockhash(&self) -> &Hash;
```

Comment:
Any reason not to include this, and few other methods immediately below, to SvmStaticMessage?

Reply:
Updated PR comment with that note